### PR TITLE
feat: separate tags into depts and active

### PR DESCRIPTION
**Pull request by ashleysyg**
update collaboration tags and schema

### CODE REVIEW

File: our-work/collaborations/borton.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 title: Borton Lab
 id: borton-lab
 description: The Borton Lab is designing, developing, and deploying neurotechnology to better understand the nervous system and improve human lives. CCV has supported software development of Behavioral Tasks used in clinical studies that can be deployed to multiple platforms as well as to the browser. CCV Software Engineers have also supported the Open Mind Consortium, which the Borton Lab is part of, with the goal of designing and developing open source software elements for neural sensing at home paired with peripheral monitors, and streamlined regulatory pathway for FDA approval of investigational protocols.
-tags:
-  - active
+active: True
+department:
```
Comment (mirestrepo) [inline our-work/collaborations/borton.yml:5]: for tags in this one `software-engineering`

File: our-work/collaborations/borton.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 title: Borton Lab
 id: borton-lab
 description: The Borton Lab is designing, developing, and deploying neurotechnology to better understand the nervous system and improve human lives. CCV has supported software development of Behavioral Tasks used in clinical studies that can be deployed to multiple platforms as well as to the browser. CCV Software Engineers have also supported the Open Mind Consortium, which the Borton Lab is part of, with the goal of designing and developing open source software elements for neural sensing at home paired with peripheral monitors, and streamlined regulatory pathway for FDA approval of investigational protocols.
-tags:
-  - active
+active: True
```
Comment (mirestrepo) [inline our-work/collaborations/borton.yml:4]: Not active anymore

File: our-work/collaborations/creton.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 title: The Creton Laboratory
 id: creton-lab
 description: Research in the Creton laboratory is focused on brain development and behavior, using zebrafish as a model system. The studies aim to provide a better understanding of basic biological mechanisms and to contribute to the prevention and treatment of various human disorders, including visual impairment, developmental brain disorders and Alzheimer's disease. CCV's collaboration with Creton Lab is to automate the tracking of zebrafish behaviors with the help of deep learning models and developing tools to analyze those behaviors.
-tags:
-  - active
+active: True
```
Comment (mirestrepo) [inline our-work/collaborations/creton.yml:4]: @ashokrags is this still active?

File: our-work/collaborations/creton.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+department:
   - neuro
 investigators:
```
Comment (mirestrepo) [inline our-work/collaborations/creton.yml:7]: tags: software-engineering, deep-learning, DeepLabCut

File: our-work/collaborations/dennery.yml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 title: Dennery Lab
 id: dennery-lab
 description: The Dennery lab investigates neonatal lung injury and repair. CCV's collaboration with the Dennery lab involved analyzing and publishing ChIP-Seq and single cell RNA-seq data.
-tags:
-  - active
+active: True
+department:
   - computational-biology
```
Comment (mirestrepo) [inline our-work/collaborations/dennery.yml:6]: @ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?

File: our-work/collaborations/fedulov.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
 title: Fedulov Lab
 id: fedulov-lab
 description: The Fedulov lab investigates epigenetic regulation. CCV's collaboration with the Fedulov lab involved analyzing and publishing reduced-representation bisulfite sequencing data.
-tags:
-  - active
+active: True
```
Comment (mirestrepo) [inline our-work/collaborations/fedulov.yml:4]: @ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?

File: our-work/collaborations/frickel.yml [hunk 7]
```diff
@@ -1,7 +1,8 @@
 title: Institute at Brown for Environment and Society (IBES)
 id: frickel
 description: IBES supports research to understand the interactions between natural, human and social systems. CCV has supported software development and computer vision and NLP pipeline design to automate extraction of hazardous sites information from historic city directories and sandborn maps
-tags:
+active: False
+department:
   - sociology
```
Comment (mirestrepo) [inline our-work/collaborations/frickel.yml:6]: tags: computer-vision, nlp

File: our-work/collaborations/hurt.yml [hunk 8]
```diff
@@ -1,7 +1,8 @@
 title: Hurt Lab
 id: hurt-lab
 description: The Hurt Lab focuses on the creation of 2D and 3D nanomaterials and their effects on human health and the environment.  CCV has helped the hurt lab with the design and implementation of a database to track toxicity of 2D nanomaterials.
-tags:
+active: False
+department:
   - engineering
 people:
```
Comment (mirestrepo) [inline our-work/collaborations/hurt.yml:7]: tags: data-management

File: our-work/collaborations/kantor.yml [hunk 9]
```diff
@@ -1,8 +1,8 @@
 title: Kantor Lab
 id: kantor-lab
 description: The Kantor lab HIV/AIDS reseach includes the evolution of antiretroviral resistance, treatment monitoring, and transmission, with a focus on HIV/AIDS locally and in resource-limited settings. CCV collaborates with the Kantor lab on transmission inference related projects.
-tags:
-  - active
+active: True
+department:
```
Comment (mirestrepo) [inline our-work/collaborations/kantor.yml:5]: @ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?

File: our-work/collaborations/mcdonald.yml [hunk 10]
```diff
@@ -1,8 +1,8 @@
 title: McDonald Lab
 id: mcdonald-lab
 description: Dr. McDonald's primary research focus is on improving maternal and infant health in resource poor settings, with particular emphasis on the role of the placenta in mediating the impact of maternal infectious disease on fetal development. CCV is collaborating with Dr. McDonald on applying metagenomics methods to study the impact of schistosomiasis on the breast milk and placental microbiome.
-tags:
-  - active
+active: True
+department:
```
Comment (mirestrepo) [inline our-work/collaborations/mcdonald.yml:5]: @ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?

File: our-work/collaborations/mchugh.yml [hunk 11]
```diff
@@ -1,7 +1,8 @@
 title: Patrick McHugh
 id: mchugh-lab
 description: Dr. McHugh's research interests focus on legitimacy and network theory in the context of social media and decision making. CCV collaborated with Dr. McHugh on research looking into tipping points using logistic regression and machine learning.
-tags:
+active: False
+department:
   - engineering
```
Comment (mirestrepo) [inline our-work/collaborations/mchugh.yml:6]: tags: data-science

File: our-work/collaborations/peac.yml [hunk 12]
```diff
@@ -1,8 +1,8 @@
 title: Psychiatry, Embodiment and Computation Lab (PEAC)
 id: peac-lab
 description: The PEAC Lab examines computational basis of our perception, learning, and control of our bodies and the external world. CCV supports the software architecting and development of their SOMA mobile app
-tags:
-  - active
+active: True
+department:
```
Comment (mirestrepo) [inline our-work/collaborations/peac.yml:5]: tags: software-engineering

File: schemas/collaborations.yml [hunk 13]
```diff
@@ -48,18 +49,18 @@ tag: "enum(
             'applied-math',
             'system-administration'
           )"
+tag: "enum(
+            'system-administration'
```
Comment (mirestrepo) [inline schemas/collaborations.yml:53]: add the suggested tags above

File: our-work/collaborations/vci.yml [hunk 14]
```diff
@@ -1,11 +1,11 @@
...
 description: The Venture Capital Inclusion (VCI) Lab uses data to identify levers for change towards a more inclusive venture capitalist (VC) industry. CCV's collaboration with the VCI Lab and Crunchbase focuses on examining how VC decsion-making and networks impact the flow of resources in entrepreneurial ecosystems.
-tags:
-  - active
+active: True
+department:
   - finance
 investigators:
   - name: Banu Ozkazanc-Pan, PhD
 people:
   - name: Matt Slivinski
```
[the remark this answers is not in the document]
Comment (ashleysyg) [reply our-work/collaborations/vci.yml:10]: remove the file altogether or change to inactive and remove people?

[the remark this answers is not in the document]
Comment (mirestrepo) [reply our-work/collaborations/vci.yml:10]: Just remove people and change to inactive

File: our-work/collaborations/burke.yml [hunk 15]
```diff
@@ -1,12 +1,12 @@
...
-tags:
-  - active
+active: True
+department:
   - computer-vision
 investigators:
   - name: Taylor Burke, Ph.D.
   - name: Thomas Serre, Ph.D.
 people:
   - name: Matt Slivinski
```
[the remark this answers is not in the document]
Comment (mirestrepo) [reply our-work/collaborations/burke.yml:11]: remove people altogether. this one is also inactive.

**Review by mirestrepo — APPROVED** [review]
looks great! I just had one suggestion for changing Borton from active to inactive, but otherwise, from my point of view, this is ready to merge!

**Pull request merged**